main() -> main {
    error("Huh? It's error function.")
} main;
